test/integration/user_checks_out_from_cart_test.rb

```diff
@@ -16,6 +16,13 @@ class UserChecksOutFromCartTest < ActionDispatch::IntegrationTest
       click_button "Login"
     end
 
+    StripeMock.start
+    customer = Stripe::Customer.create(
+    {
```

Comment:
Redundant curly braces around a hash parameter.<br>Indent the first parameter one step more than the start of the previous line.